src/odemis/driver/technolution.py

```diff
@@ -0,0 +1,544 @@
+import copy
+import json
+import queue
+from datetime import datetime
+import logging
+import time
+import threading
+import requests
+import numpy
+from odemis import model
+import weakref
+
+from odemis import model
+from src.openapi_server.models.mega_field_meta_data import MegaFieldMetaData
+from src.openapi_server.models.cell_parameters import CellParameters as CellAcqParameters
+from src.openapi_server.models.field_meta_data import FieldMetaData
+from src.openapi_server.models.error_status import ErrorStatus
+
+from odemis.model import _vattributes
+
+_METHOD_GET = 1
+_METHOD_POST = 2
+
+
+class AcquisitionServer(model.HwComponent):
+
+    def __init__(self, name, role, server_URL, children={}, deamon=None, **kwargs):
+        super(AcquisitionServer, self).__init__(name, role, **kwargs)
+
+        self.server_URL = server_URL
+        # TODO K.K. Set external storage and connection
+
+        # Stop any acquisition if already one was in progress
+        self.simpleASMCall(self.server_URL + "/scan/finish_mega_field", _METHOD_POST, 204)
+
+        # TODO K.K. remove *kwargs and include setup of entire component in try (each sperated) first check if childeren
+        # holds value of component
+        try:
+            kwargs = children["MirrorDescanner"]
+        except Exception:
+            raise ValueError("Required child MirrorDescanner not provided")
+        self._MirrorDescanner = MirrorDescanner("MirrorDescanner", role=None, parent=self)
+        self.children.value.add(self._MirrorDescanner)
+
+        try:
+            kwargs = children["EBeamScanner"]
+        except Exception:
+            raise ValueError("Required child EBeamScanner not provided")
+        self._EBeamScanner = EBeamScanner("EBeamScanner", role=None, parent=self)
+        self.children.value.add(self._EBeamScanner)
+
+        try:
+            kwargs = children["MPPC"]
+        except Exception:
+            raise ValueError("Required child MPPC not provided")
+        self._MPPC = MPPC("MPPC", None, self)
+        self.children.value.add(self._MPPC)
+
+    @classmethod
+    def simpleASMCall(cls, url, method, expected_status, data=None, raw_response=False, timeout=600):
+        """
+
+        :param url: url of the command, server part is defined in global variable URL
+        :param method: getting or posting via global variables _METHOD_GET/_METHOD_POST
+        :param expected_status: expected feedback of server for a positive call
+        :param data: data added to the call
+        :param raw_response: specified the format of the structure returned
+        :param timeout: [s] if within this period no bytes are received an timeout exception is raised
+        :return: status_code(posting), content dictionary(getting), or entire response (raw_response=True)
+        """
+        logging.getLogger().setLevel(logging.INFO)
```

Comment:
Don't set the log level here.

Reply:
I like to output in the log if a call is made (and/or sucessfull). But outputting it as a warning doesn't seem fit. What do you think? Don't ouput this in the log? Because this method can be called also outside the class defining it at the start of the file doesn't work always I'm afraid.

Reply:
It is good that you log the call, but I would leave setting the log level up to the user. Usually, we set the log level in the odemis.config file if I remember correctly. I would suggest using logging.debug to log your message.